packages/superflare/tests/scheduled.test.ts

```diff
@@ -0,0 +1,87 @@
+import { describe, expect, test, vi } from "vitest";
+import { Task, shouldRunTask } from "../src/scheduled";
+
+describe("shouldRunTask", () => {
+  test("runs every minute by default", () => {
+    const task = new Task(vi.fn());
+
+    expect(shouldRunTask(task, new Date("2021-01-01T00:00:00"))).toBe(true);
+  });
+
+  test("#everyMinute", () => {
+    const task = new Task(vi.fn()).everyMinute();
+
+    expect(shouldRunTask(task, new Date("2021-01-01T00:00:00"))).toBe(true);
```

Comment:
I am just confused about using an old date like `new Date("2021-01-01T00:00:00")` and returning `true` I expect it to be an invalid date because all tasks scheduled should be in the future

Reply:
So the logic here is pretty simple and only checks the current minute/hour/date/etc. 

This is designed to check the constraints of the passed value [`event.scheduledTime`](https://developers.cloudflare.com/workers/runtime-apis/scheduled-event/#properties) which is when the schedule was meant to run originally. It's possible that there could be a backlog of cron events on Cloudflare's side, and if you evaluated only the current time instead, it's possible you'd miss certain event intervals.